### PR TITLE
Test zen1

### PR DESCRIPTION
**Pull request by wfeldt**
zenworks image adjustments

### CODE REVIEW

File: data/root/zenroot.file_list [hunk 1]
```diff
@@ -51,6 +51,7 @@ else
   openSUSE-release-mini:
 endif
 
+?grub:
```
Comment (jreidinger) [inline data/root/zenroot.file_list]: why grub1?

**Comment by jreidinger** [comment]
otherwise it looks good. Just grub1 do not make sense for me

File: data/root/zenroot.file_list (hunk 1 above)
Comment (wfeldt) [inline data/root/zenroot.file_list]: they still use it

Comment (jreidinger) [inline data/root/zenroot.file_list]: OK, I hope they do not expect that we will support it.

**Comment by jreidinger** [comment]
LGTM

**Pull request merged**